src/styles/blocks/_calendar.scss

```
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  width: $calendar_width;
+  padding: 10px;
+  flex-wrap: wrap;
+
+  @for $index from 1 through $month-length {
+    &--month-length--#{$index} :nth-child(n + #{$index+1}) {
+      display: none;
+    }
+  }
```

```suggestion
  @for $index from 28 through $month-length {
    &--month-length--#{$index} :nth-child(n + #{$index+1}) {
      display: none;
    }
  }
```
Можно начать с 28, чтобы не создавать лишние стили, так как месяц не может быть короче 27 дней